brushfire-core/src/main/scala/com/stripe/brushfire/Tree.scala

```diff
@@ -1,14 +1,14 @@
 package com.stripe.brushfire
 
+import com.stripe.bonsai.FullBinaryTreeOps
 import com.twitter.algebird._
 
-// type Tree[K, V, T] = AnnotatedTree[K, V, T, Unit]
```

Comment:
So it actually looks like type alias still exists in `package.scala`. I'm not sure why that comment is there -- I'm going to remove it.